third-party/realdds/src/dds-participant.cpp

```diff
@@ -209,6 +210,10 @@ dds_participant::qos::qos( std::string const & participant_name )
     //udp_transport->maxMessageSize = 1470; TODO this affects reading, too! We need writer-only property for this...
     transport().use_builtin_transports = false;
     transport().user_transports.push_back( udp_transport );
+
+    // If the user has many local interfaces (possible: I have 5!) then anything over the limit will not get
+    // communicated and end-point discovery will be affected. We want to avoid this so increase the limit:
+    allocation().locators.max_unicast_locators = 10;
```

Comment:
Use this as default but we may need a parameter for this to allow users connecting many cameras to increase the number

Reply:
You can override - see dds-serialization.cpp line 729